packages/core-modules/contracts/mixins/CoreCommsMixin.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 
 import "@synthetixio/core-contracts/contracts/proxy/ProxyStorage.sol";
 
-contract CommsMixin is ProxyStorage {
+contract CoreCommsMixin is ProxyStorage {
```

Comment:
Please, confirm that's the best place and name for this mixin to be part of core-modules

Reply:
Location looks good. Just a nit maybe, rename to `CommsMixin` since people won't probably extend it, so no need for the `Core` word?